src/pages/kb/user-guide/querying/writing-queries.md

```diff
@@ -60,18 +63,27 @@ By default each query starts as an unpublished draft, which means that:
 To publish a query, give it a name or click the `Publish` button. It's also possible to unpublish a published query by clicking on the `Unpublish` button in the query menu.
 
 
-
-
-
-
-### Archiving a Query
+## Archiving a Query
 
 Once a query is no longer useful, you can archive it. Archiving is almost the same as deleting, except that **direct links to the query will still work.** To archive a query, open the little menu at the top-right area of the query editor, next to the Save button and click Archive.
 
 ![](/assets/images/docs/gitbook/archive_query.png)
 
-### Duplicating (Forking) a Query
+## Duplicating (Forking) a Query
 
 If you need to create a copy of an existing query (created by you or someone else), you can fork it. To fork a query, just click on the Fork button (see example below)
 
 ![](/assets/images/docs/gifs/queries/fork_query.gif)
+
+## Managing Query Permissions
+
+By default, saved queries can only be modified by the user who created them and members of the Admin group. But Redash includes experimental support to share edit permissions with non-Admin users. You need to enable it first. Open your organization settings and check the "Enable experimental multiple owners support"
```

Comment:
Worth mentioning that only an admin can enable this feature.

Reply:
Fixed.